torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -201,21 +203,107 @@ def __init__(self, gm, skip_constructors=False):
         # see: [constant folding refining of symints]
         self.node_replacements_shapes: Dict[torch.fx.Node, List[int]] = {}
 
+        self.symint_nodes = _SymHashingDict()
+
+        # reference from torch/_funtorch/partitioners.py:get_default_op_list
```

Comment:
We need tags for these lol

Reply:
Yea https://github.com/pytorch/pytorch/issues/129020..